### ISSUE TITLE
Alter table wizard hangs when MariaDB table uses system versioning

### ISSUE THREAD

**Issue by stefanfrings2** [webyog/sqlyog-community#2255]
Steps to reproduce:
```
CREATE TABLE `test` (
  id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
  product VARCHAR(80)
) WITH SYSTEM VERSIONING;
```
Now right-click on the table name and and select "alter table". A wizard dialog appear. Klick on the "SQL Preview" tab. Now SqlYog hangs, although I did not modify anything.

The alter table statement works fine when executed manually:
```
SET @@system_versioning_alter_history = KEEP;
ALTER TABLE test CHANGE product product VARCHAR(100);
```
Even after setting this system property, the wizard dialog hangs anyway.

Environment:
Debian Linux 10
MariaDB 10.3.22
Wine 4.0-2
SqlYog Community 13.1.5 64bit

**Comment by sibwy** [comment]
Thank you for bringing this to our notice. We can reproduce this issue with SQLyog v13.1.6 on Windows OS as well.

**Comment by sibwy** [comment]
Another user reported the issue. User report can be found in SF >> 00862855

**Comment by kanugoyal** [comment]
Automation Details:
We have automated the test cases for this ticket and execution is completed. Please find the attached Ranorex log report.
Build: SQLyog-13.1.8-6
[Issue_2255_log.zip](https://github.com/webyog/sqlyog-community/files/6263951/Issue_2255_log.zip)

**Comment by sibwy** [comment]
Fixed in [v13.1.8](https://blog.sqlyog.com/sqlyog-mysql-gui-13-1-8-release/).
